tests/integration_tests/sqllab_tests.py

```diff
@@ -406,22 +407,19 @@ def test_search_query_on_text(self):
         self.assertEqual(2, len(data))
         self.assertIn("birth", data[0]["sql"])
 
-    def test_search_query_on_time(self):
+    def test_search_query_filter_by_time(self):
         self.run_some_queries()
         self.login("admin")
         first_query_time = (
             db.session.query(Query).filter_by(sql=QUERY_1).one()
         ).start_time
         second_query_time = (
-            db.session.query(Query).filter_by(sql=QUERY_3).one()
+            db.session.query(Query).filter_by(sql=QUERY_2).one()
         ).start_time
-        # Test search queries on time filter
-        from_time = "from={}".format(int(first_query_time))
-        to_time = "to={}".format(int(second_query_time))
```

Comment:
Bycatch: this test is flaky. Since the original code `int(...)` basically rounded down the float epoch, we actually queried for `QUERY_1` + `QUERY_2`---even though `second_query_time` filtered by `QUERY_3`. But this still sometimes fail because the rounded down time can be anywhere between QUERY_1 start time and QUERY_3 start time.